tests/extensions/zip/fixtures/obj_tree.json

```diff
@@ -0,0 +1,169 @@
+{
+    "test_file_tree": [
```

Comment:
Here is an example of a fully built `obj_tree`. Due to the requirement of the library we use, the tree is a list which contains only the root node: `[root,]`. Each node in the tree is a dictionary with four keys: `icon`, `text`, `data` and `children`. `text` is the file/folder's name and `children` is a list of child nodes.